src/rdf4cpp/rdf/datatypes/registry/DatatypeRegistry.cpp

```diff
@@ -2,6 +2,10 @@
 
 namespace rdf4cpp::rdf::datatypes::registry {
 
+#ifndef RDF4CPP_DBPEDIA_MODE
+bool dbpedia_mode = false;
```

Comment:
rename to something like
- relaxed_parsing_mode
- lax_parsing_mode

or similar